### PR TITLE
Added logo and minor style adjustments

### PR DESCRIPTION
**Pull request by ChrisCogswell**

### CODE REVIEW

File: dashboard.html [hunk 1]
```diff
@@ -24,44 +26,49 @@
...
                       <div class="col-md-8">
                         <div class="card-body">
                           <h5 class="card-title">Launch Info</h5>
-                          <h4 class="card-text">Countdown:</h4>
-                          <p class="card-text">Location:</p>
-                          <p class="card-text">Date/Time: <span id="launch-start"></span></p>
-                          <p class="card-text">PAYLOAD:</p>
+                          <h4 class="card-text" id="countdown">Countdown:</h4>
+                          <p class="card-text" id="launch-location">Location:</p>
+                          <p class="card-text" id="launch-date">Date/Time</p>
```
Comment (johnsickels) [inline dashboard.html]: switch this with `<p class="card-text">Date/Time: <span id="launch-start"></span></p>`, it currently works with one of the javascript api pulls

**Review by johnsickels — CHANGES_REQUESTED** [review]
Just need the one line corrected and the rest is glorious! Can't wait to see it, good work

File: dashboard.html (hunk 1 above)
Comment (ChrisCogswell) [reply dashboard.html]: Went ahead and changed the rest of the id's to spans as well

**Review by johnsickels — APPROVED** [review]

**Comment by johnsickels** [comment]
Super excite

**Pull request merged**